### ISSUE TITLE
performance?

### ISSUE THREAD

**Issue by Apostatee** [Stick-To/Object-Detection-API-Tensorflow#9]
Your codes are so neatly and beautifully built, have you tested their performances? I wonder whether they can achieve the same results in original paper? Thanks for sharing these codes.

**Comment by zzundark** [comment]
I think all model needs pretrained weights...if model is training without them, the training is very slow...so I hope release backbone weights or pretrained weights in google drive(it is so hard to download model's weights in baidu cloud)

**Comment by Stick-To** [comment]
I haven't the exact performance of these model, I think these model are easer to understand

**Comment by Stick-To** [comment]
@zzundark I don't have the pretrained weight of these model, I release these model just because they are easer to understand.
The examples of results could be seen in corresponding repository in https://github.com/Stick-To

**Comment by Stick-To** [comment]
And I have no sufficent computing power to provide all pretrained model

**Comment by zzundark** [comment]
> And I have no sufficent computing power to provide all pretrained model

Hmm...how could you train your models? and what training set do you achieve model's examples of results in corresponding repository?

**Comment by Stick-To** [comment]
It only the results of the voc, not other lager dataset

**Comment by zzundark** [comment]
> It only the results of the voc, not other lager dataset

If you don't mind, can you provide all model's weights that training voc dataset?

**Comment by Stick-To** [comment]
Very regret, they are deleted

**Comment by Apostatee** [comment]
I trained the VOC2007 with centernet，  loss did not converge and always turned to ‘’NAN‘’。The learning rate I set was 0.0001

**Comment by Stick-To** [comment]
@Apostatee Sorry,  I have no pretrained weights to privided you, I release this code just because it is easer for people to understand comparing with official code. If you want to get great performance, maybe you could try official code.
